packages/embeds/embed-core/src/embed.ts

```diff
@@ -779,7 +849,6 @@ document.addEventListener("click", (e) => {
   api("modal", {
     calLink: path,
     config,
-    uid: modalUniqueId,
```

Comment:
`modal` would now compute its Id itself and store on Cal instance instead of storing on DOM element